src/custom.ts

```diff
@@ -0,0 +1,55 @@
+import { EngineeringNotation } from "./engineering";
+import Decimal from "break_infinity.js/break_infinity";
+import { toEngineering } from "./utils";
+
+export class CustomNotation extends EngineeringNotation {
+  private letters: string | string[];
+  private mantissaExponentSeparator: string;
+  private separator: string;
+  
+  constructor(letters: string | string[], mantissaExponentSeparator="", separator="") {
```

Comment:
- Validate parameters before `super` call
- Validate input parameters, not `this.parameter`
- Use more formal error messages, these are usually easier to get right
- Spaces

```suggestion
  constructor(letters: string | string[], mantissaExponentSeparator = "", separator = "") {
    if (letters.length < 2) {
      throw new Error("The supplied letter sequence must contain at least 2 letters");
    super();
    this.letters = letters;
    this.mantissaExponentSeparator = mantissaExponentSeparator;
    this.separator = separator;
  }
```